build/Shared/Utf8JsonStreamReader.cs

```diff
@@ -234,7 +246,7 @@ internal bool ReadNextTokenAsBoolOrThrowAnException(byte[] propertyName)
             {
                 throw new ArgumentException(
                     string.Format(CultureInfo.CurrentCulture,
-                    Strings.Invalid_AttributeValue,
+                    PublicStrings.Invalid_AttributeValue,
```

Comment:
If we duplicate the resource and the name, I don't think we'd need the public strings. We'd just need every project that includes this to have the same Strings name. Or is the namespace a problem?

Reply:
Yes, if the resources are compiled into each assembly, they're under that namespace:
![image](https://github.com/user-attachments/assets/bc564d21-4b62-4840-916f-66c51760662c)
So each assembly would need another `.resx` like `SharedStrings` with a common namespace so that shared code could use it.